gr-utils/python/modtool/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright 2011 Free Software Foundation, Inc.
+# Copyright 2018 Free Software Foundation, Inc.
```

Comment:
When modifying a file, don't replace the year but add it to keep track of its changes

`# Copyright 2011, 2018 Free Software Foundation, Inc.`

This goes for all mods on the Copyright year that you made

Reply:
The files that are newly created will have only 2018. All the `core` scripts and the `tools` scripts have a previous copyright intact.